ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_mathematical_functions.py

```diff
@@ -2819,6 +2819,45 @@ def test_jax_remainder(
     )
 
 
+@handle_frontend_test(
+    fn_tree="jax.numpy.roots",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=1,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=3,
+    ),
+)
+def test_jax_roots(
+    *,
+    dtype_and_x,
+    test_flags,
+    on_device,
+    fn_tree,
+    frontend,
+    backend_fw,
+):
+    input_dtype, x = dtype_and_x
+
+    def call():
+        helpers.test_frontend_function(
+            input_dtypes=input_dtype,
+            test_flags=test_flags,
+            frontend=frontend,
+            backend_to_test=backend_fw,
+            fn_tree=fn_tree,
+            on_device=on_device,
+            p=x[0],
+            atol=1e-05,
+            rtol=1e-03,
+        )
+        ret, frontend_ret = call()
```

Comment:
unless test_values=False, the test_frontend_function doesn't return the values. In addition this `call()` should be made out side the the definition of `call`. Tbh we don't even need to define it as a call. But thats fine.
Once this is updated pls request my review. Thnx